packages/react-query/src/shared/hooks/deprecated/createHooksInternal.tsx

```diff
@@ -291,6 +293,34 @@ function __createHooksInternal<
     TPath extends keyof TQueryValues & string,
     TQueryFnData = TQueryValues[TPath]['output'],
     TData = TQueryValues[TPath]['output'],
+  >(
+    pathAndInput: [path: TPath, ...args: inferHandlerInput<TQueries[TPath]>],
+    opts: DefinedUseTRPCQueryOptions<
+      TPath,
+      TQueryValues[TPath]['input'],
+      TQueryFnData,
+      TData,
+      TError
+    >,
+  ): DefinedUseTRPCQueryResult<TData, TError>;
+  function useQuery<
+    TPath extends keyof TQueryValues & string,
+    TQueryFnData = TQueryValues[TPath]['output'],
+    TData = TQueryValues[TPath]['output'],
+  >(
+    pathAndInput: [path: TPath, ...args: inferHandlerInput<TQueries[TPath]>],
+    opts?: UseTRPCQueryOptions<
+      TPath,
+      TQueryValues[TPath]['input'],
+      TQueryFnData,
+      TData,
+      TError
+    >,
+  ): UseTRPCQueryResult<TData, TError>;
+  function useQuery<
+    TPath extends keyof TQueryValues & string,
+    TQueryFnData,
+    TData,
```

Comment:
This is v9 which is deprecated and will be deleted (see #3496)

Better to leave it untouched.

```suggestion
```

Reply:
Nice!!